content/en/docs/releases/supported-releases/index.md

```diff
@@ -64,7 +64,7 @@ As of now, data plane to data plane is compatible across all versions; however,
 | master          | No, development only |                   |                        |                               |                                          |
 | 1.17            | Yes                  | February 14, 2023 | ~Sept 2023 (Expected)  | 1.23, 1.24, 1.25, 1.26        | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21, 1.22 |
 | 1.16            | Yes                  | November 15, 2022 | ~June 2023 (Expected)  | 1.22, 1.23, 1.24, 1.25        | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21       |
-| 1.15            | Yes                  | August 31, 2022   | ~March 2023 (Expected) | 1.22, 1.23, 1.24, 1.25        | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21       |
+| 1.15            | Yes                  | August 31, 2022   | Mar 28, 2023           | 1.22, 1.23, 1.24, 1.25        | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21       |
```

Comment:
This is handled by #12919 along with the EOL announcement.

Reply:
@dhawton 
I see. 
Then this PR seems unnecessary, so I close it.